templates/template_parser.toml

```diff
@@ -463,11 +463,6 @@
     description = ""
     ref = "Y/N/NK"
 
-  [visit.treatment_indication_anticoagulation]
```

Comment:
I see this has been removed from the template parser. Has it been removed from the visit schema file?